src/amber/cli/templates/app/src/views/layouts/application.ecr.ecr

```diff
@@ -12,7 +12,7 @@
     <div class="masthead">
       <div class="container">
         <nav class="nav">
-          <%="<"%>%= render(partial: "layouts/_nav.<%= @language %>") %>
+          <%="<"%>%= render(partial: "layouts/_nav.ecr") %>
```

Comment:
Would this work for `slang` templates?

Reply:
This is in the ecr template so we can be specific here.